Makefile

```diff
@@ -134,6 +134,7 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 .PHONY: deploy
 deploy: manifests kustomize kind load-on-kind ## Deploy controller to the K8s cluster specified in ~/.kube/config.
 	cd config/frr-k8s && $(KUSTOMIZE) edit set image controller=${IMG}
+	$(KUBECTL) -n frr-k8s-system delete ds frr-k8s-daemon || true
```

Comment:
not: why not run `undeploy` before instead?

Reply:
because undeploy does a lot more, removes the CRDs for example. Here what I care about is a scenario where we change the images and want to reload. Am I missing some scenario where it would make sense to do otherwise?

Reply:
nope you're right